### DISCUSSION TITLE
Releasing the Final Update of the Year '23

### DISCUSSION THREAD

**Discussion by aolle** [WebGoat/WebGoat#1659] [General]
Hello @zubcevic , @nbaars ,

As we approach the end of the year, I wanted to inquire about the feasibility of launching the last release for this year. Given that a considerable number of issues have been addressed and resolved, it seems opportune to consider this as an option.

In my specific case, I am planning to expand and introduce new lessons, but with a targeted release date set for the beginning of the upcoming year.

What do you think?

Regards.

**Comment by zubcevic** [comment]
Yes sounds like a great idea. Also do we want to get rid of the version 8 mentions on the Readme?

**Comment by aolle** [comment]
Yes, I think so... right? like in our release notes. WDYT?

**Comment by aolle** [comment]
Thank you for the release.
